src/mongo/db/storage/wiredtiger/encryption_keydb.cpp

```diff
@@ -179,9 +179,10 @@ void EncryptionKeyDB::init_masterkey() {
         } else {
             encoded_key = kmipReadKey(_kmipMasterKeyId);
             if (encoded_key.empty()) {
-                throw std::runtime_error(
-                    "Cannot start. Master encryption key is absent in KMIP. Check "
-                    "configuration options.");
+                LOGV2_FATAL_NOTRACE(29110,
```

Comment:
Don't forget to update [LOGV2 codes](https://confluence.percona.com/display/PSMDB/Document+logging+subsystem+identifiers+used+in+Percona+code).